cif_core.dic

```diff
@@ -9756,6 +9756,44 @@ save_exptl_absorpt.process_details
 
 save_
 
+save_exptl_absorpt.special_details
+
+    _definition.id                '_exptl_absorpt.special_details'
+
+    loop_
+      _alias.definition_id
+         '_exptl_absorpt_special_details'
+         '_exptl.absorpt_special_details'
+
+    _definition.update            2022-11-22
+    _description.text
+;
+    Details of the absorption correction process applied to the
+    measured intensities that cannot otherwise be given by other
+    _exptl_absorpt.* data names.
+;
+    _name.category_id             exptl_absorpt
+    _name.object_id               special_details
+    _type.purpose                 Describe
+    _type.source                  Recorded
+    _type.container               Single
+    _type.contents                Text
+
+    loop_
+      _description_example.case
+;
+       X-ray beam inhomogeneity and goniometer imperfection account, by and
+       large, for much of the difference between estimated values of Tmin and
+       Tmax and the _exptl_absorpt_correction_ values.
+;
+;
+       X-ray linear absorption coefficient is wavelength dependent. The value
+       of 0.420 (mm^-1^) shown in _exptl_absorpt_coefficient_mu is the sample
+       absorption coefficient for a wavelength of 1.5 \%A.
+;
```

Comment:
The provided examples look good in my eyes, but in case "live" examples are preferred, I attach a file with the `_exptl_absorpt_special_details` data item values observed in the COD. Some are clearly not fit as examples, but there are definitely more than a few more meaningful ones. Each line in the file starts with a COD ID, followed by a tab and the data item value that was unfolded into a single line. In case you decide to go with any of these examples, I can easily provide references to the original CIF publication which could potentially be included into the example loop using the `_description_example.detail` data item (this seems to sometimes be done in examples, though, I am unsure if this is necessary in this case).

File with the values: [special-detail-values-from-the-COD.txt](https://github.com/COMCIFS/cif_core/files/10108594/special-detail-values-from-the-COD.txt)

Reply:
I think the examples currently provided are adequate to demonstrate ways in which this data name are used, so there is no need to for further examples. I think longer more detailed examples are useful when showing how a whole category, or several categories, interact.

Reply:
Mine are kind of already "live", inasmuch as they were heavily "inspired" by real ones. By "reference", do you mean to the journal paper where the data originally came from? I'll see what I can find.